docs/docs/development/setup.md

```diff
@@ -331,13 +331,33 @@ Since the Zephyr™ SDK is not available for Windows, we recommending following
 </TabItem>
 <TabItem value="mac">
 
-#### GNU ARM Embedded
+#### Zephyr™ SDK Bundle
 
-Since the Zephyr™ SDK is not available for macOS, we recommending following the steps to install the GNU ARM Embedded toolchain:
+:::note
+It is recommended to extract the Zephyr™ SDK Bundle at one of these locations:
 
+- `~/`
+- `~/.local`
+- `~/.local/opt`
+- `~/bin`
+- `/opt`
+- `/usr/local`
+
+Note that the install includes the directory `zephyr-sdk-X.XX.X`
+:::
+
+Download the [latest Zephyr SDK™ Bundle](https://github.com/zephyrproject-rtos/sdk-ng/releases) to a directory of your choice. If the host architecture is Apple Silicon based, download the aarch64 version.
```

Comment:
For official docs, I'd recommend we link to a static, known-good version like we do for other operating systems.

Reply:
I think I fixed it, let me know if I can improve upon it!